# 历史馆
history_museum: 
  position: 
    x: 2.74470472336
    y: -0.159480959177
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.0353931596443
    w: 0.999373465853

# 自然馆
nature_museum: 
  position: 
    x: 2.63857078552
    y: -1.15717506409
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.0141917797499
    w: 0.999899291623

# 陶瓷展览
ceramics_exhibition: 
  position: 
    x: -0.595248103142
    y: -0.0851126760244
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.999373464239
    w: -0.0353932051983

# 端砚艺术展览
duanyan_art_exhibition: 
  position: 
    x: -0.598946750164
    y: -1.05250120163
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.99908166902
    w: -0.042846454089

# 充电桩
charging_post: 
  position: 
    x: 0.681133151054
    y: -0.406202316284
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.014574797654
    w: 0.999893781996

# 起始点
init_position: 
  position: 
    x: 0.681133151054
    y: -0.406202316284
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.014574797654
    w: 0.999893781996
